syntax = "proto3";

message SensorData {
    string device_id = 1;
    int32 event_id = 2;
    float humidity = 3;
    float temperature = 4;
}
